de.dlr.sc.virsat.uiengine.ui/src/de/dlr/sc/virsat/uiengine/ui/editor/GenericEditor.java

```diff
@@ -587,7 +587,7 @@ protected void addPages() {
 		}
 
 		Diagnostic diagnostic = editingDomain.getResourceSet().getResourceToDiagnosticsMap().get(resource);
-		boolean hasErrors = diagnostic != null && diagnostic.getSeverity() == Diagnostic.ERROR;
+		boolean hasErrors = diagnostic != null && diagnostic.getSeverity() != Diagnostic.OK;

```

Comment:
Just for clarity: According to the new code I think we would stay on the error page in the case of a warning.
Is this intended?

Reply:
Yes this is intended. We want to react to warnings as well. It is also not staying on the page, but prominently showing it.

Reply:
Alright